### PR TITLE
Es6

### PR DESCRIPTION
**Pull request by josuerushanika**
In this project, l restructure my  Awesome books app code. The goal is to make it more organized by using modules. I also practice the ES6.

**Project requirements**

_Use ES6 syntax._
Divide my code into modules. Save each module in a separate file in the modules dir. Import modules in the index.js file.
Refactor my  methods and functions to arrow functions.
Use let and const in a correct way.

_Display the current date below the navigation bar._

I used a library [Luxon](https://moment.github.io/luxon/#/install?id=es6)

_Project overview_

image

### CODE REVIEW

File: modules/navigation.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  sec2.classList.add('no-display');
+  contactBtn.classList.add('active-btn');
+  addbookBtn.classList.remove('active-btn');
+  listBtn.classList.remove('active-btn');
+};
+// contactBtn.addEventListener('click', showContact);
+// listBtn.addEventListener('click', listbookShow);
+// addbookBtn.addEventListener('click', addNewBookShow);
+
+// export default navLink;
```
Comment (victorokoroji) [inline modules/navigation.js]: - [ ] KIndly remove all commented code from your code base so that your code will be clean  and readable

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @josuerushanika, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/navigation.js (hunk 1 above)
Comment (josuerushanika) [reply modules/navigation.js]: Thx it is fixed

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @josuerushanika ✔️, 

You have done a good job implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by josuerushanika** [comment]
Thank You

**Pull request merged**